tmt/steps/report/html.py

```diff
@@ -44,19 +48,20 @@ class ReportHtml(tmt.steps.report.ReportPlugin):
     _keys = ["open"]
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Optional[str] = None) -> List[click.Option]:
```

Comment:
I believe we should use the `ClickOptionDecoratorType` introduced in 262a87c92cd1d0ef4bb8977704069163d296973a by @thrix.

Reply:
Right, fixed.